### PR TITLE
let admins always remove users

### PR DESCRIPTION
**Pull request by unicornzero**

### CODE REVIEW

File: tests/integration/components/organizations/user-card-test.js [hunk 1]
```diff
@@ -314,55 +271,6 @@ describe('Integration: UserCard', function() {
...
-        let transitionToStub;
-
-        beforeEach(function() {
-          transitionToStub = sinon.stub();
-          stubService(this, 'router', 'router', {
-            transitionTo: transitionToStub,
-          });
-        });
-
-        it('requests confirmation and calls destroyRecord', async function() {
```
Comment (cadeParade) [inline tests/integration/components/organizations/user-card-test.js]: This test still seems relevant?

**Comment by unicornzero** [comment]
I've been thinking about edge cases a lot and will probably want to show extra messaging if someone is removing a user from an SSO org. So, I'm going to hold off on merging this today as some of the code and test setup may carry over.
